meta-xt-prod-cockpit-rcar-driver-domain/recipes-core/systemd/files/70-eth0.network

```diff
@@ -0,0 +1,9 @@
+[Match]
```

Comment:
Hi @iusyk, please, provide evidence that this file is mentioned somewhere in base layers as SRC_URI and FILES.

Reply:
./meta-rcar-gen3-adas/recipes-core/systemd/systemd/70-eth0.network

Reply:
Hi @iusyk, thanks for the clarification. This topic is closed.